lib/bulk/ordered.js

```diff
@@ -114,6 +114,11 @@ class OrderedBulkOperation extends BulkOperationBase {
    */
   execute(_writeConcern, options, callback) {
     const ret = this.bulkExecute(_writeConcern, options, callback);
+
+    if (!(ret && ret.options && ret.callback)) {
```

Comment:
I think this might not be a complete enough check: what if `options` is `null`/`undefined`?